### ISSUE TITLE
ui: buttons to select-only or all-except each time series

### ISSUE THREAD

**Issue by flyingmutant** [VKCOM/statshouse#209]
`Shift-click` is completely undiscoverable.

**Comment by leeoniya** [comment]
https://github.com/leeoniya/uPlot/blob/master/dist/uPlot.d.ts#L278

**Comment by flyingmutant** [comment]
> https://github.com/leeoniya/uPlot/blob/master/dist/uPlot.d.ts#L278

Yep, we want to try to make it obvious for all users that we support both modes, because now only the power-users are aware about the second behaviour you get when holding `Shift`.
